README.md

```diff
@@ -1,6 +1,6 @@
-# @storacha/elizaos-storacha
+# @storacha/elizaos-adapter
 
-Storacha implements a database adapter for [ElizaOS](https://elizaos.github.io/eliza/) that enables decentralized storage of agent data using the Storacha network.
+This project implements a database adapter for [ElizaOS](https://elizaos.github.io/eliza/) that enables decentralized storage of agent data using the Storacha network. Which facilitates data sharing and data redundancy - all data is persisted in Filecoin L1.
```

Comment:
Ah - i meant this for under "Key Features of the Storacha Adapter" in `integration.md`.